h/search/core.py

```diff
@@ -34,14 +35,25 @@ class Search(object):
     :type stats: statsd.client.StatsClient
     """
     def __init__(self, request, separate_replies=False, stats=None, _replies_limit=200):
-        self.request = request
-        self.es = request.es
+        self.es_index = request.es.index
```

Comment:
Note we said goodbye to the request.es object.

Reply:
Can you explain how there would be unintentional divergence? I could see that that would be the case if we were using both es.conn and elasticsearch-dsl connect, but I don’t see how that would be the case if we were using one or the other. They are only configured in one location AFAIK.